src/main.c

```diff
@@ -18,7 +24,44 @@ static void repl(VM* vm) {
       break;
     }
 
-    interpret(vm, line);
+    int lBraceCount = 0;
+    int rBraceCount = 0;
+
+    for (int i = 0; line[i]; i++) lBraceCount += (line[i] == '{');
+    for (int i = 0; line[i]; i++) rBraceCount += (line[i] == '}');
+
+    if (lBraceCount > rBraceCount) {
+      char code[1024];
+
+      memcpy(code, line, sizeof(line));
+
+      for (;;) {
+        printf(".. ");
```

Comment:
Change the `..` to match the other indicator. Can you also change it to indicate how deep we currently are. ie:

```js
> global
>> outer block
>>> nested block
and so on...
```

Reply:
I see. Do you want me to do smth like:
```c
// Not sure if the args will be same but 
static void showBlock(VM* vm, char* indicator) {
    if (lBraceCount != 1 && lBraceCount > rBraceCount) { showBlock(args...) }
}